.github/workflows/actions/configuration/action.yaml

```diff
@@ -48,29 +48,13 @@ runs:
       docker images ls -a
     shell: bash
 
-  - name: Download Artifact Apiserver
-    uses: actions/download-artifact@v2
-    with:
-      name: apiserver_img
-      path: /tmp
-
-  - name: Load KubeRay Apiserver Docker Image
-    run: |
-      docker load --input /tmp/apiserver.tar
-      docker images ls -a
-    shell: bash
-
   - name: Run configuration tests for sample YAML files.
     # compatibility test depends on operator & apiserver images built in previous steps.
     run: |
-      pushd manifests/base/
-      kustomize edit set image kuberay/operator=kuberay/operator:${{ steps.vars.outputs.sha_short }}
-      kustomize edit set image kuberay/apiserver=kuberay/apiserver:${{ steps.vars.outputs.sha_short }}
-      popd
```

Comment:
Except here and [compatibility/action.yaml](https://github.com/ray-project/kuberay/pull/759/files/829195b2bd5fff6b29e849b7fae32bd3146cc89a#diff-88197d1eee4d640617cb62d8517a0490a2406e4248d9b14214047d299f61a7fe), does kustomize be utilized in other places? If not, I think we can remove kustomize dependency in [ray-operator/DEVELOPMENT.md](https://github.com/ray-project/kuberay/blob/master/ray-operator/DEVELOPMENT.md).

Reply:
Good catch! I will update it.

Reply:
Update https://github.com/ray-project/kuberay/pull/759/commits/78849d69ed70f4c9456ec4b6340ccb07d9296ed4